.readthedocs.yaml

```diff
@@ -0,0 +1,22 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required. Represents Configuration file v2.
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
+
+# Build documentation in the "docs/" directory with Sphinx, Not manually set in the project.
+#sphinx:
+#  configuration: docs/conf.py
+
+# Optional but recommended, declare the Python requirements required
+# to build your documentation
```

Comment:
newline is not necessary here since we have agreed on line length of 120 (instead of 80).